src/azure/policyHelper.ts

```diff
@@ -138,10 +172,47 @@ function getPolicyDefinition(definitionPath: string): any {
     }
   }
 
-  validateDefinition(definition, definitionPath);
+  validatePolicy(definition, definitionPath, FRIENDLY_DEFINITION_TYPE);
   return definition;
 }
 
+function getPolicyInitiative(initiativePath: string): any {
+  const policySetPath = path.join(initiativePath, POLICY_SET_FILE_NAME);
+  const policySetParametersPath = path.join(initiativePath, POLICY_SET_PARAMETERS_FILE_NAME);
+  const policySetDefinitionsPath = path.join(initiativePath, POLICY_SET_DEFINITIONS_FILE_NAME);
+
+  let initiative = getFileJson(policySetPath);
+
+  if ((!initiative.properties || !initiative.properties.policyDefinitions) && doesFileExist(policySetDefinitionsPath)) {
```

Comment:
if (!initiative) ???
 
if (!initiative.properties)  initiative.properties = {};

if(!initiative.properties.policyDefinitions) populatePolicySetDefinitions(initiative);

if(!initiative.properties.parameters) populatePolicySetParameters(initiative);

Fetch the path in respective methods